README.md

```diff
@@ -144,110 +144,10 @@ To get latest updates on the project and to interact with the community. We are
-    </td>
-    <td align="center">
-      <a href="https://github.com/Faisalhs51">
-        <img src="https://github.com/Faisalhs51.png" width="100px;" alt="username2"/><br />
-        <sub><b>Faisalhs51</b></sub>
-      </a>
-    </td>
-    <td align="center">
-      <a href="https://github.com/akssshh">
-        <img src="https://github.com/akssshh.png" width="100px;" alt="username2"/><br />
-        <sub><b>akssshh</b></sub>
-      </a>
-      </td>
-  </tr>
-  <tr>  
-  <!--    2nd row        -->
-   <td align="center">
-      <a href="https://github.com/ASIMNEUPANE">
-        <img src="https://github.com/ASIMNEUPANE.png" width="100px;" alt="username3"/><br />
-        <sub><b>Asimneupane</b></sub>
-      </a>
-    </td>    
-   <td align="center">
-      <a href="https://github.com/Nishanth019">
-        <img src="https://github.com/Nishanth019.png" width="100px;" alt="username3"/><br />
-        <sub><b>Nishanth</b></sub>
-      </a>
-    </td>    
-   <td align="center">
-      <a href="https://github.com/Iamdivyak">
-        <img src="https://github.com/Iamdivyak.png" width="100px;" alt="username3"/><br />
-        <sub><b>Divya</b></sub>
-      </a>
-    </td>
-    <td align="center">
-      <a href="https://github.com/vivek-singh-13">
-        <img src="https://github.com/vivek-singh-13.png" width="100px;" alt="username2"/><br />
-        <sub><b>vivek singh</b></sub>
-      </a>
-    </td>
-    <td align="center">
-      <a href="https://github.com/AditiSingh2003">
-        <img src="https://github.com/AditiSingh2003.png" width="100px;" alt="username2"/><br />
-        <sub><b>Aditi Singh</b></sub>
-      </a>
-      </td>
-  </tr>
-  <!--    2nd row        -->
-   <td align="center">
-      <a href="https://github.com/Himangshi">
-        <img src="https://github.com/Himangshi.png" width="100px;" alt="username3"/><br />
-        <sub><b>Himangshi</b></sub>
-      </a>
-    </td>    
-   <td align="center">
-      <a href="https://github.com/singhaman09">
-        <img src="https://github.com/singhaman09.png" width="100px;" alt="username3"/><br />
-        <sub><b>Aman Singh</b></sub>
-      </a>
-    </td>    
-   <td align="center">
-      <a href="https://github.com/kartik2433">
-        <img src="https://github.com/kartik2433.png" width="100px;" alt="username3"/><br />
-        <sub><b>kartik</b></sub>
-      </a>
-    </td>
-    <td align="center">
-      <a href="https://github.com/ani-netizen">
-        <img src="https://github.com/ani-netizen.png" width="100px;" alt="username2"/><br />
-        <sub><b>Anirudh</b></sub>
-      </a>
-    </td>
-    <td align="center">
-      <a href="https://github.com/pranshu-5123">
-        <img src="https://github.com/pranshu-5123.png" width="100px;" alt="username2"/><br />
-        <sub><b>Pranshu</b></sub>
-      </a>
-      </td>
-  </tr>
-</table>
-
+<a href="https://github.com/abhay-raj19/FitBody/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=abhay-raj19/FitBody" />
```

Comment:
but this image that is embedded automatically generates the image for the contribuors section and then what is the need for adding that workflow in the github/workflow :(

Reply:
@abhay-raj19 

yes, That method was to add contributor profiles by defining the workflow of the yml file. But  using that code it only it makes a rectangular profile . so I changed the way of implementing. 
ya that was my mistake I have deleted the workflow.yml files now.
please have a look on it now